pkg/service/service.go

```diff
@@ -51,11 +51,12 @@ func Search(client *occlient.Client, name string) ([]string, error) {
 }
 
 // CreateService creates new service from serviceCatalog
-func CreateService(client *occlient.Client, serviceName string, serviceType string, applicationName string) error {
+func CreateService(client *occlient.Client, serviceName string, serviceType string, servicePlan string, parameters []string, applicationName string) error {
```

Comment:
just a doubt, are the additional parameters ok to be mandatory?
Asking only bcoz we didn't have them before, so should we also accommodate the case of not passing it from the perspective of config generated in occlient? Please ignore if handled

Reply:
They are mandatory otherwise you can't bind next the service to your application. Ideally, they should be added to the application definition to be less verbose

Example

```
services:
  - class: dh-postgresql-apb
    name: my-postgresql-db
    plan: dev
    parameters:
      - name: postgresql_user
        value: luke
      - name: postgresql_password
        value: secret
      - name: postgresql_database
        value: my_data
      - name: postgresql_version
        value: 9.6
```